content/chef_compliance_phase.md

```diff
@@ -0,0 +1,314 @@
++++
+title = "Compliance Phase"
+draft = true
+
+aliases = ["/chef_compliance_phase.html"]
+
+[menu]
+  [menu.infra]
+    title = "Compliance Phase"
+    identifier = "chef_infra/features/chef_compliance_phase.md Compliance Phase"
+    parent = "chef_infra/features"
+    weight = 80
++++
+
+[\[edit on GitHub\]](https://github.com/chef/chef-web-docs/blob/master/content/chef_compliance_phase.md)
+
+
+The Compliance Phase allows you to run Chef InSpec profiles as part of a Chef Infra client run. It downloads configured profiles from various sources like Chef Automate, Chef Supermarket or git, and reports audit runs to Chef Automate.
+
+## Overview
+
+### Component Architecture
+```
+ ┌──────────────────────┐    ┌──────────────────────┐    ┌─────────────────────┐
+ │  Chef Infra Client   │    │  Chef Infra Server   │    │    Chef Automate    │
+ │                      │    │      (optional)      │    │                     │
+ │ ┌──────────────────┐ │    │                      │    │                     │
+ │ │                  │◀┼────┼──────────────────────┼────│  Profiles           │
+ │ │ Compliance Phase │ │    │                      │    │                     │
+ │ │                  │─┼────┼──────────────────────┼───▶│  Reports            │
+ │ └──────────────────┘ │    │                      │    │                     │
+ │                      │    │                      │    │                     │
+ └──────────────────────┘    └──────────────────────┘    └─────────────────────┘
+```
+
+Chef InSpec Profiles can be hosted from a variety of locations:
+
+```
+ ┌──────────────────────┐                                ┌─────────────────────┐
+ │  Chef Infra Client   │     ┌───────────────────────┐  │    Chef Automate    │
+ │                      │  ┌──│ Profiles(Supermarket, │  │                     │
+ │ ┌──────────────────┐ │  │  │ Github, local, etc)   │  │                     │
```

Comment:
```suggestion
 │ ┌──────────────────┐ │  │  │ GitHub, local, etc)   │  │                     │
```

Reply:
I think I can enable Mermaid for diagramming in Hugo. Would this improve your lives? https://learn.netlify.app/en/shortcodes/mermaid/

Reply:
These diagrams seem to be describing "in one scenario you use a proxy/middleman" which is not really worth a lot of effort to preserve. I'll just delete the diagrams.